### PR TITLE
Update the wording

### PR DESCRIPTION
**Pull request by queenypingcap**

### CODE REVIEW

**Comment by hicqu** [comment]
LGTM. @andelf , PTAL.

File: sql/json-functions-generated-column.md [hunk 1]
```diff
@@ -7,18 +7,18 @@ category: compatibility
 
 ## 概述
 
-为了在功能上与最新版的MySQL（版本 5.7 及以上）保持同步，同时更好地支持文档存储类业务，我们在最新版本的 TiDB 中加入了 JSON 的支持。用户可以在 TiDB 的表中使用 JSON 类型的字段，同时以生成列（generated column）的方式为 JSON 文档内部的字段建立索引。基于此，用户可以很灵活地处理那些 schema 不确定地业务，同时不必受限于传统文档数据库糟糕的读性能及匮乏的事务支持。
+为了在功能上兼容 MySQL 5.7 及以上，同时更好地支持文档类型存储，我们在最新版本的 TiDB 中加入了 JSON 的支持。TiDB 所支持的文档是指以 JSON 为编码类型的键值对的组合。用户可以在 TiDB 的表中使用 JSON 类型的字段，同时以生成列（generated column）的方式为 JSON 文档内部的字段建立索引。基于此，用户可以很灵活地处理那些 schema 不确定的业务，同时不必受限于传统文档数据库糟糕的读性能及匮乏的事务支持。
 
 ## JSON功能介绍
 
-TiDB的 JSON 主要参考了 MySQL 5.7 的用户接口。例如，可以创建一个表，包含一个JSON字段来存储那些复杂的信息：
+TiDB的 JSON 主要参考了 MySQL 5.7 的用户接口。例如，可以创建一个表，包含一个 JSON 字段来存储那些复杂的信息：
```
Comment (andelf) [inline sql/json-functions-generated-column.md]: a space after TiDB

File: sql/json-functions-generated-column.md [hunk 2]
```diff
@@ -37,15 +37,13 @@ INSERT INTO person (name, address_info) VALUES ("John", CAST('123' AS JSON));
...
+现在，如果我们想查询表中所有居住在北京的用户，该怎么做呢？需要把数据全拉回来，然后在业务层进行过滤吗？不需要，和 MongoDB 等文档数据库相同，我们有在服务端支持用户各种复杂组合查询条件的能力。你可以这样写 SQL：
 
 ```sql
 SELECT id, name FROM person WHERE JSON_EXTRACT(address_info, '$.city') = 'Beijing');
 ```
 
-`JSON_EXTRACT` 是TiDB支持的一个函数，熟悉 MySQL 5.7 的 JSON 功能的朋友应当很熟悉，它们的用法完全相同。
-这个函数的意思就是，从 `address_info` 这个文档中取出名为city这个字段。它的第二个参数是一个“路径表达式”，
-我们由此可以指定到底要取出哪个字段。关于路径表达式的完整语法描述比较复杂，我们还是看几个简单的例子来了解它大致的使用吧：
+TiDB 支持 `JSON_EXTRACT` 函数，该函数与 MySQL 5.7 中 `JSON_EXTRACT`的用法完全相同。这个函数的意思就是，从 `address_info` 这个文档中取出名为 `city` 这个字段。它的第二个参数是一个“路径表达式”，我们由此可以指定到底要取出哪个字段。关于路径表达式的完整语法描述比较复杂，我们还是通过几个简单的例子来了解其用法：
```
Comment (andelf) [inline sql/json-functions-generated-column.md]: a space after `JSON_EXTRACT`

File: sql/json-functions-generated-column.md [hunk 3]
```diff
@@ -68,41 +66,41 @@ SELECT JSON_EXTRACT(@person,  '$.friends[2].name'); -- gets NULL
...
-熟悉 MySQL 5.7 的用户会发现，TiDB 尚未完全支持所有 MySQL 5.7 中的 JSON 函数。这是因为我们在实现的时候，一期目标是能够提供完备的 **MySQL X Plugin** 支持即可，而这已经涵盖大部分常用的JSON增删改查的功能了。更多的函数功能，只要社区有需求，我们会继续全力支持的。
+熟悉 MySQL 5.7 的用户会发现，TiDB 尚未完全支持所有 MySQL 5.7 中的 JSON 函数。这是因为我们的一期目标是能够提供完备的 **MySQL X Plugin** 支持即可，而这已经涵盖大部分常用的 JSON 增删改查的功能了。如有需要，我们会继续完善对其他函数的支持。
 
-## 使用生成列对JSON建索引
+## 使用生成列对 JSON 建索引
 
-在有了上述的知识铺垫后，数据库的老司机们一定会发现，我们在查询 JSON 中的一个字段时，走的是全表扫描啊，不开心！的确，使用 TiDB 的 ``EXPLAIN`` 语句时，一个比 MySQL 完备得多的结果会告诉我们，确实是全表扫描了。那么，我们能否对 JSON 字段进行索引呢？
+在有了上述的知识铺垫后，您可能会发现我们在查询 JSON 中的一个字段时，走的是全表扫描。使用 TiDB 的 `EXPLAIN` 语句时，一个比 MySQL 完备得多的结果会告诉我们，的确是全表扫描。那么，我们能否对 JSON 字段进行索引呢？
 
 首先，这种索引姿势是错误的：
```
Comment (andelf) [inline sql/json-functions-generated-column.md]: Is 索引姿势 suitable here?

oral or formal

**Comment by shenli** [comment]
LGTM

**Pull request merged**